src/main/java/org/broadinstitute/hellbender/tools/spark/sv/discovery/SvDiscoverFromLocalAssemblyContigAlignmentsSpark.java

```diff
@@ -152,6 +154,17 @@ protected void runTool(final JavaSparkContext ctx) {
         dispatchJobs(ctx, contigsByPossibleRawTypes, svDiscoveryInputMetaData, assemblyRawAlignments, writeSAMFiles);
     }
 
+    private String getOutputPrefixWithSampleNameAndTrailingUnderscore() {
```

Comment:
Couldn't you just call this method `getOutputPrefix`? That's a very long name.

Reply:
renamed as suggested and added doc